.github/workflows/cpu-ci.yml

```diff
@@ -35,6 +35,8 @@ jobs:
         python -m pip install tensorflow
         # required for ensemble tests
         python -m pip install xgboost
+        # required for feast unit tests
+        python -m pip install "feast<0.20"
```

Comment:
I think we'd also have to add this to the stuff that gets installed on the Jenkins machine

Reply:
If I've pulled the thread correctly, it looks like it's already there: https://github.com/NVIDIA-Merlin/Merlin/blob/a1cc48fe23c4dfc627423168436f26ef7e028204/ci/dockerfile.ci#L18